src/plugins/intel_cpu/src/plugin.cpp

```diff
@@ -784,22 +898,29 @@ InferenceEngine::IExecutableNetworkInternal::Ptr Engine::ImportNetwork(std::istr
 
     CNNNetwork cnnnetwork;
     deserializer >> cnnnetwork;
-
+ 
     Config conf = engConfig;
-    conf.readProperties(config);
 
     // import config props from caching model
     auto function = cnnnetwork.getFunction();
-    if (function->has_rt_info("intel_cpu_hints_config") && !conf.perfHintsConfig.ovPerfHint.empty()) {
-        const auto mode_name = conf.perfHintsConfig.ovPerfHint;
-        if (mode_name == CONFIG_VALUE(LATENCY) || mode_name == CONFIG_VALUE(THROUGHPUT)) {
-            const auto& hints_config = function->get_rt_info<ov::AnyMap>("intel_cpu_hints_config");
-            const auto hints_param_name = mode_name + "_" + std::string(ov::num_streams.name());
-            const auto it = hints_config.find(hints_param_name);
-            if (it != hints_config.end()) {
-                conf.readProperties({{std::string(ov::num_streams.name()), it->second.as<std::string>()}});
-            } else {
-                IE_THROW() << "Cache file doesn't contain precalculated number of streams for mode " << mode_name;
+    if (is_cpu_map_available()) {
+        LoadMultiThreadingConfig(conf, function);
```

Comment:
What is the technical justification for necessety to store/load all multithreading config during export/import? In other words why should LoadNetwork and ImportNetwork routines have different behavior?

Reply:
The different routines between LoadNetwork and ImportNetwrok is legacy behavior. 

Since we need initial CPU map and schedule streams in both LoadNetwork and ImportNetwork, would check if we can merge them in single routine. TODO JIRA CVS-112767 is created.

Reply:
Why do we need this WA then? Are we fixing smt urgent?
Again my point is ther shouldn't be any differences between `LoadNetwork` and `ImportNetwrok` pipelines in terms of multithreading set up.
The only exception is storage of model prefered num streams al long as we are using common MemBandwidthPressureTolerance function which recognize only common ngraph opset, while model loaded from cache contains cpu specific operations. So the solution is to store already computated value within the model. However other multithreading parameters don't have such dependency and I would prefer to avoid its introduction.